lib/tasks/scrape_paws_data.rake

```diff
@@ -0,0 +1,64 @@
+task({ :scrape_paws_data => :environment}) do
+  Pet.destroy_all
+
+  url = "https://www.pawschicago.org/our-work/pets-adoption/pets-available"
+  webpage = HTTP.get(url)
+  parsed_page = Nokogiri::HTML(webpage.body.to_s)
+  pet_links = parsed_page.css('.adopt-pet a')
+
+  links = []
+    
+  pet_links.each do |pet_link|
+    links.push(pet_link.attributes.fetch("href").to_s)
+  end
+
+  links.each do |link|
+    pet_url = "https://www.pawschicago.org#{link}"
+    pet_webpage = HTTP.get(pet_url)
+    pet_parsed_page = Nokogiri::HTML(pet_webpage.body.to_s)
+
+    pet = Pet.new
+    pet.name = pet_parsed_page.css('.aqua-text').children.to_s
+    pet.adoption_center_identifier = link
+    pet.children_competability_ranking = if pet_parsed_page.css('.children .rating_default').children.to_s == "UNKNOWN" || pet_parsed_page.css('.children .rating_default').children.to_s == ""
+        then nil 
```

Comment:
The `then` keyword is optional is the expression (`pet_parsed_page.css('.children .rating_default').children.to_s == "UNKNOWN" || pet_parsed_page.css('.children .rating_default').children.to_s == ""` in this case) is separated by a newline. 

See more [here](https://www.tutorialspoint.com/ruby/ruby_if_else.htm)